### PR TITLE
ResultTable on protobuf metadata

### PR DESCRIPTION
**Pull request by denisgursky**
- ResultTable will use protobuf metadata if available and fallback to json otherwise.
- Most of the work happens in `resultUtils.ts` specifically in `getTypeDefFromProtobuf`
- All standard types, specialization on all standard types, all standard types in a value type are covered with integration tests. See `tests.ts`

### CODE REVIEW

File: src/results/resultUtils.ts [hunk 1]
```diff
@@ -212,7 +221,130 @@ export function getTypeDef(type: string): RelTypeDef {
...
+
+      return {
+        type: 'Constant',
+        value: {
+          ...typeDef,
+          value,
+        } as RelTypedValue,
+      };
+    } else {
+      const value = unflattenContantValue(
```
Comment (NRHelmi) [inline src/results/resultUtils.ts:250]: I think this is a typo `unflattenConstantValue` ?

**Review by NRHelmi — APPROVED** [review]
Awesome, LGTM thanks @denisgursky

File: src/results/resultUtils.ts (hunk 1 above)
Comment (denisgursky) [reply src/results/resultUtils.ts]: Hehe, I stared for one minute and couldn't see it :D Thx!

**Comment by vilterp** [comment]
🎉

**Pull request merged**